pep8_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find . -type f -name "*.py" ! -name 'conf.py' | xargs flake8 --max-line-length=100
```

Comment:
Any special reason you chose to put this line in a shell script? I am not against it, but adding 1 line to `.travis.yaml` save you a 3-line new file.